parsers/FR_O_old.py

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/env python3
```

Comment:
If FR_O_old.py is in the parser folder it will be run when we schedule the parsers. I'm not sure this is the intended behavior. Better to place it under parsers/archived if it's no longer going to be used.

Reply:
Oh right, that was accidental. I think I'll just remove it as it has too many "estimation" features and tries to split unknown into other modes.